src/runtime/components/NuxtTurnstileNuxtScript.vue

```diff
@@ -0,0 +1,79 @@
+<script setup lang="ts">
+
+import { nextTick, useScriptCloudflareTurnstile, useRuntimeConfig, useNuxtApp, ref, onMounted, onBeforeUnmount } from '#imports'
+
+const props = defineProps({
```

Comment:
We want to expose a `trigger` option to allow users granular access of when the script loads, see https://github.com/nuxt/scripts/blob/main/src/runtime/components/ScriptGoogleMaps.vue#L44